### DISCUSSION TITLE
Version calculation and release branches

### DISCUSSION THREAD

**Discussion by dominikjeske** [GitTools/GitVersion#2659] [Q&A] [answered]
I have a question about calculation algorithm. We would like to version our projects basing on tag prefix but as soon as there is any branch release/version it is overriding the version we set on tags for each projects. I know that monorepo is not supported scenario but I’m wonder is it possible to turn off versioning basing on current release branches in repo? I figure out that setting is-release-branch: false on release branch configuration do the trick but I’m not sure if this change could affect other use cases.

**Comment by asbjornu** [comment]
Setting `is-release-branch` to `false` should have the effect you're asking for. Have you tried it?

**Reply by dominikjeske** [reply]
Yes I tried - It is described in last sentence. It works but I was wondering is it a right way to do.

**Reply by asbjornu** [reply, marked as answer]
If you don't want the branch name to affect the version number, then yes – that's the right way to do it.
